src/main/java/core/basesyntax/dao/jdbc/DaoJdbcCarImpl.java

```diff
@@ -0,0 +1,171 @@
+package core.basesyntax.dao.jdbc;
+
+import core.basesyntax.dao.DaoCar;
+import core.basesyntax.exceptions.DataProcessingException;
+import core.basesyntax.lib.Dao;
+import core.basesyntax.model.Car;
+import core.basesyntax.model.Driver;
+import core.basesyntax.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@SuppressWarnings("unused")
```

Comment:
what exactly you are suppressing here?

Reply:
My classes. IDEA saying that they are not used because my usage is through reflection injection so I suppressed it to not have warnings every time I'm pushing.